CLI/Compile.cpp

```diff
@@ -46,10 +46,9 @@ struct GlobalOptions
     int optimizationLevel = 1;
     int debugLevel = 1;
 
-    std::string vectorLib;
-    std::string vectorCtor;
-    std::string vectorType;
-
+    const char* vectorLib;
+    const char* vectorCtor;
+    const char* vectorType;
```

Comment:
```suggestion
    const char* vectorLib = nullptr;
    const char* vectorCtor = nullptr;
    const char* vectorType = nullptr;
```
In case it's moved from implicitly null globals to a different scope.